packages/report-server/src/reportBuilder/transform/functions/mergeRows/mergeRows.ts

```diff
@@ -20,32 +20,36 @@ type MergeRowsParams = {
   where: (parser: TransformParser) => boolean;
 };
 
-const paramsValidator = yup.object().shape({
+const optionalMergeStrategyNameValidator = yup
+  .mixed<keyof typeof mergeStrategies>()
+  .oneOf(Object.keys(mergeStrategies) as (keyof typeof mergeStrategies)[]);
+const mergeStrategyNameValidator = optionalMergeStrategyNameValidator.required();
+
+export const paramsValidator = yup.object().shape({
   groupBy: starSingleOrMultipleColumnsValidator,
-  using: yup.lazy((value: unknown) => {
-    const optionalMergeStrategyNameValidator = yup
-      .mixed<keyof typeof mergeStrategies>()
-      .oneOf(Object.keys(mergeStrategies) as (keyof typeof mergeStrategies)[]);
-    if (value === undefined) {
-      return optionalMergeStrategyNameValidator;
-    }
+  using: yupUtils.lazy(
+    (value: unknown) => {
+      if (value === undefined) {
+        return optionalMergeStrategyNameValidator;
+      }
 
-    const mergeStrategyNameValidator = optionalMergeStrategyNameValidator.required();
-    if (typeof value === 'string') {
-      return mergeStrategyNameValidator;
-    }
+      if (typeof value === 'string') {
+        return mergeStrategyNameValidator;
+      }
 
-    if (typeof value === 'object' && value !== null) {
-      const mergeStrategyMapValidator = Object.fromEntries(
-        Object.entries(value).map(([columnName]) => [columnName, mergeStrategyNameValidator]),
-      );
-      return yup.object().shape(mergeStrategyMapValidator);
-    }
+      if (typeof value === 'object' && value !== null) {
+        const mergeStrategyMapValidator = Object.fromEntries(
+          Object.entries(value).map(([columnName]) => [columnName, mergeStrategyNameValidator]),
+        );
+        return yup.object().shape(mergeStrategyMapValidator);
+      }
 
-    throw new yup.ValidationError(
-      'mergeUsing must be either a single merge strategy, or a mapping between columns and merge strategies',
-    );
-  }),
+      throw new yup.ValidationError(
+        'mergeUsing must be either a single merge strategy, or a mapping between columns and merge strategies',
+      );
+    },
+    [optionalMergeStrategyNameValidator, yup.object().shape({})],
```

Comment:
We need to pass in all validators from lazy function (`mergeStrategyNameValidator` and `optionalMergeStrategyNameValidator` are the same, so only pass in one of them), so that we can convert them into json schema.

Reply:
If that requires more explanation, we can have a chat on it.